benches/get_chat_msgs.rs

```diff
@@ -31,6 +31,8 @@ fn criterion_benchmark(c: &mut Criterion) {
             b.to_async(AsyncStdExecutor)
                 .iter(|| get_chat_msgs_benchmark(black_box(&path.as_ref()), black_box(&chats)))
         });
+    } else {
+        println!("env var not set: DELTACHAT_BENCHMARK_DATABASE");
```

Comment:
+1, iirc, i was also stumbling upon missing DELTACHAT_BENCHMARK_DATABASE for some other benchmark at some point :)